resources/tests/test_api.py

```diff
@@ -30,21 +30,21 @@ def test_1_create_index(self):
 
     def test_2_video_docs(self):
         doc_list = api.get_video_docs()
-        self.assertTrue(doc_list is not None and len(doc_list) > 10)
+        self.assertEqual(len(doc_list), 12)
 
     def test_3_video_items(self):
         video_items = api.get_video_items()
-        self.assertTrue(len(video_items.get('video_list', '')) > 15)
+        self.assertEqual(len(video_items.get('video_list', '')), 15)
```

Comment:
Here, the same criteria should be applied as with the others; the video list is website dependent and its quantity is expected to vary along the time (as it actually did between your changes). As it's sole purpose is to test that a significant valid list of videos is retrieved, testing a quantity above 10 (i.e. 10 videos + the "next page" entry) should be enough. Using an exact match assert type will led to failed error tests in future commits.

So please, consider to change the line like this:

`self.assertTrue(len(video_items.get('video_list', '')) > 10)`

This line should be the only one you would modify in this commit.

Thanks!